FileReader.py

```diff
@@ -0,0 +1,12 @@
+import os
+
+class FileReader:
```

Comment:
This is good, but let's take this in another direction.
I want to treat this module as a general utilities module, which means there's no need for a class. You can define a general method called `read_file` that receives a file path and returns the content of the file.
Rename the module accordingly.